documentation/openapi.json

```diff
@@ -1255,137 +1153,104 @@
   },
   "components": {
     "schemas": {
-      "ValidationProblemDetails": {
+      "CollectionRuleDescription": {
```

Comment:
Everything from here to the end is just a reordering (in alphabetical order) of the schemas, except it thinks that TimeSpan.Ticks is not readonly anymore even though it is a getter-only property.